src/test/analyzer_test.ts

```diff
@@ -168,6 +168,21 @@ suite('Analyzer', () => {
           ['polymer.html', 'foo.js', 'foo.css']);
     });
 
+    test('polymer css import scanner', async() => {
+      let contents = `<html><head>
+          <link rel="import" type="css" href="bar.css">
```

Comment:
Oh, crumbs. 😨   It hadn't occurred to me that I had to be a bit *smarter* about this.  Putting thinking cap back on, looking up documentation for the feature...

According to https://www.polymer-project.org/1.0/docs/devguide/styling#external-stylesheets the link should live in the `<dom-module>` but outside the `<template>`.

I expect this is because `<template>` contents are parsed but not "active" or what-not.  On that note, do we intend that the analyzer flag or exclude certain features during scan when they are encountered inside <template> tags for this reason?